README.md

```diff
@@ -1,7 +1,94 @@
 # sonar-scanner-python
 A wrapper around SonarScanner CLI, available on PyPI.
 
-# Installation 
+# Installation
+
+Install with pip:
+```
+pip install sonar-scanner-python
+```
+
+# Usage 
+
+## Configuration 
+
+There are two ways of setting up sonar-scanner-python.
+
+### with a pyproject.toml file
```

Comment:
I think the suggested approach to use the `tool.sonar` key makes sense. However I'm also a bit uneasy that we document this before actually making the decision and implementing it (what if we end up doing differently? or forgoing reading properties from `pyproject.toml` files in the first release?). 

That said I don't challenge the approach and I'm confident we can implement it as described so I wouldn't advocate for removing this section, but maybe here as well we could have a `FIXME` which we could link to the ticket associated with implementing this (so that whoever implements it can check the implementation and documentation actually match).

Reply:
I guess I did some things in the wrong order. I think it is a good discussion to have on what/how should we implement those kind of things. I will add the fix me and change a bit the related tickets so that we don't get confused.